### PR TITLE
Update Skinning Contest #2 Wiki Page

### PR DESCRIPTION
**Pull request by RockRoller01**
Really hope i didnt make a mistake with the flags.. would be great if someone could check

Currently is missing the podium image, can not be merged before this is added. Also needs to wait until newspost is out

## Self-check

- [X] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

File: wiki/Contests/Skinning_Contest/2/en.md [hunk 1]
```diff
@@ -75,9 +108,20 @@ The Skinning Contest #2 is run by various community members.
...
+[flag_MX]: /wiki/shared/flag/MX.gif "Mexico"
+[flag_MY]: /wiki/shared/flag/MY.gif "Malaysia"
+[flag_NL]: /wiki/shared/flag/NL.gif "Netherlands"
+[flag_PH]: /wiki/shared/flag/PH.gif "Philippines"
+[flag_PL]: /wiki/shared/flag/PL.gif "Poland"
+[flag_RO]: /wiki/shared/flag/RO.gif "Romania"
+[flag_TR]: /wiki/shared/flag/TR.gif "Turkey"
+[flag_GB]: /wiki/shared/flag/GB.gif "United Kingdom"
+[flag_US]: /wiki/shared/flag/US.gif "United States"
+[flag_VN]: /wiki/shared/flag/VN.gif "Vietnam"
```
Comment (zeusminus) [inline wiki/Contests/Skinning_Contest/2/en.md:127]: These should be ordered alphabetically.

```suggestion
[flag_BR]: /wiki/shared/flag/BR.gif "Brazil"
[flag_BY]: /wiki/shared/flag/BY.gif "Belarus"
[flag_CA]: /wiki/shared/flag/CA.gif "Canada"
[flag_DE]: /wiki/shared/flag/DE.gif "Germany"
[flag_FR]: /wiki/shared/flag/FR.gif "France"
[flag_GB]: /wiki/shared/flag/GB.gif "United Kingdom"
[flag_ID]: /wiki/shared/flag/ID.gif "Indonesia"
[flag_JP]: /wiki/shared/flag/JP.gif "Japan"
[flag_MX]: /wiki/shared/flag/MX.gif "Mexico"
[flag_MY]: /wiki/shared/flag/MY.gif "Malaysia"
[flag_NL]: /wiki/shared/flag/NL.gif "Netherlands"
[flag_PH]: /wiki/shared/flag/PH.gif "Philippines"
[flag_PL]: /wiki/shared/flag/PL.gif "Poland"
[flag_RO]: /wiki/shared/flag/RO.gif "Romania"
[flag_TR]: /wiki/shared/flag/TR.gif "Turkey"
[flag_US]: /wiki/shared/flag/US.gif "United States"
[flag_VN]: /wiki/shared/flag/VN.gif "Vietnam"
```

Comment (RockRoller01) [reply wiki/Contests/Skinning_Contest/2/en.md:127]: Oh they should be ordered by their code, i did it by their name 🙃

File: wiki/Contests/Skinning_Contest/2/en.md [hunk 2]
```diff
@@ -45,12 +47,43 @@ The Skinning Contest #2 is run by various community members.
...
+| ![][flag_FR] [Corne2Plum3](https://osu/ppy/sh/users/15646039) | [Snowy Circles](https://osu.ppy.sh/community/forums/topics/1498442) |
+| ![][flag_ID] [nayuu](https://osu/ppy/sh/users/12561379) | [snowyy](https://osu.ppy.sh/community/forums/topics/1498720) |
+| ![][flag_MX] [ZiskyST](https://osu/ppy/sh/users/11844975) | [The Lost Winter](https://osu.ppy.sh/community/forums/topics/1494537) |
+| ![][flag_US] [Icysandwich](https://osu/ppy/sh/users/10961544) | [Tides of osu!](https://osu.ppy.sh/community/forums/topics/1498620) |
+| ![][flag_CA] [RUDEKA](https://osu/ppy/sh/users/13015586) | [TOWN](https://osu.ppy.sh/community/forums/topics/1484572) |
+| ![][flag_MY] [HIDOKOMORI123](https://osu/ppy/sh/users/14216890) | [WINTERMACHI](https://osu.ppy.sh/community/forums/topics/1491283) |
+
+## Podium
+
+*For full results, see the [contest page](https://osu.ppy.sh/community/contests/137).*
```
Comment (Walavouchey) [inline wiki/Contests/Skinning_Contest/2/en.md:83]:
```suggestion
*For the full results, see the [contest page](https://osu.ppy.sh/community/contests/137).*
```

File: wiki/Contests/Skinning_Contest/2/en.md [hunk 3]
```diff
@@ -45,12 +47,43 @@ The Skinning Contest #2 is run by various community members.
...
+
+![](img/podium.png "Skinning Contest #2 podium")
```
Comment (Walavouchey) [inline wiki/Contests/Skinning_Contest/2/en.md:85]: file missing

**Review by Walavouchey — CHANGES_REQUESTED** [review]

File: wiki/Contests/Skinning_Contest/2/en.md (hunk 2 above)
Comment (RockRoller01) [reply wiki/Contests/Skinning_Contest/2/en.md:83]: I simply copied this from the first contests page, i guess it should also be changed there?

Comment (Walavouchey) [reply wiki/Contests/Skinning_Contest/2/en.md:83]: yes

**Review by Walavouchey — COMMENTED** [review]
also update [the index page](https://osu.ppy.sh/wiki/en/Contests)

**Comment by Walavouchey** [comment]
Is this still waiting on the podium image?

**Comment by RockRoller01** [comment]
@Walavouchey should now be fully done with this

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**